go/service/chat_local.go

```diff
@@ -313,35 +461,33 @@ func (h *chatLocalHandler) fillSenderIDsForPostLocal(arg *keybase1.PostLocalArg)
 		return fmt.Errorf("Can't send message without a current DeviceID. Are you logged in?")
 	}
 
-	arg.MessagePlaintext.ClientHeader.Sender = gregor1.UID(uid)
-	arg.MessagePlaintext.ClientHeader.SenderDevice = gregor1.DeviceID(did)
+	arg.ClientHeader.Sender = gregor1.UID(uid)
```

Comment:
This isn't quite right, I have a patch for this but it is buried somewhere. You need to convert from hex to bytes here (which is annoying I know), the cast won't do the job.